pkg/commands/edit/add/all.go

```diff
@@ -26,9 +26,12 @@ import (
 func NewCmdAdd(fsys fs.FileSystem, v ifc.Validator, kf ifc.KunstructuredFactory) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "add",
-		Short: "Adds configmap/resource/patch/base to the kustomization file.",
+		Short: "Adds secret/configmap/resource/patch/base to the kustomization file.",
```

Comment:
The English here is a little wrong.
Feel free to improve this (and make it future-proof)
by making it shorter:

```
Adds an item to the kustomization file.
```

Reply:
(English was wrong in original text)